### DISCUSSION TITLE
Objects grouping questions

### DISCUSSION THREAD

**Discussion by LucileDT** [akiraux/Akira#530] [Ideas]
Hi! As you are working on snapping object, I have a question: do you plan to group objects?

**Comment by mbfraga** [comment]
Hi! I believe grouping is in the roadmap, and snapping should work with grouping out of the box.

**Reply by Alecaddd** [reply]
Indeed, grouping is part of the Beta release wishlist.
It will happen once we have the multi selection in place.

**Reply by LucileDT** [reply]
@mbfraga saw what I was wondering, thanks!
